shared/src/license.ts

```diff
@@ -39,13 +35,24 @@ export type LicenseStatus =
   | "SUSPENDED"
   | "REVOKED";
 
-export interface LicenseData {
+export type LicenseDetails = {
   nameAndAddress: LicenseSearchNameAndAddress;
-  completedSearch: boolean;
-  expirationISO?: string;
+  licenseStatus: LicenseStatus;
+  expirationDateISO: string | undefined;
+  lastUpdatedISO: string;
+  checklistItems: LicenseStatusItem[];
+  hasError?: boolean;
+};
+
+export type LicenseTaskID = keyof typeof taskIdToLicenseName;
+
+export type LicenseName = (typeof taskIdToLicenseName)[LicenseTaskID];
```

Comment:
This is an area that I think would be simplified by typing `taskIdToLicenseName`

Reply:
agreed